src/main/java/frc/robot/commands/TransitionRunCommand.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.StorageConstants;
+import frc.robot.subsystems.Storage;
+
+public class TransitionRunCommand extends Command {
+    private Storage storage = Storage.getInstance();
+
+    private Timer timer = new Timer();
+    private double timeToRun = StorageConstants.defaultTimeToRun;
+
+    public TransitionRunCommand() {
+        addRequirements(storage);
+    }
+
+    public TransitionRunCommand(double timeToRun) {
+        addRequirements(storage);
+        this.timeToRun = timeToRun;
+    }
```

Comment:
This code is another way to set the time to run the correct way. I think it may be a good idea to follow this format for the other commands, or follow how it is found in the AcquireNote command.

Reply:
which commands don't we do that in? This is the same way we do it in acquire command

Reply:
The difference is that you are initializing the instance field to the constant found in the StorageConstants and then changing the value in the second constructor if we need to set a custom value for this command. In the AcquireNote command, we initialize the instance of time within the constructors only. The instance field is only a declaration.